apps/mcp/linear/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@apps/linear-mcp",
+  "version": "0.1.0",
+  "private": true,
+  "description": "MCP server for basic Linear operations",
+  "license": "AGPL-3.0",
+  "type": "module",
+  "bin": "./dist/index.js",
+  "scripts": {
+    "build": "rimraf dist && tsc --build && shx chmod +x dist/*.js",
+    "dev": "chokidar 'dist/main.js' -c 'chmod +x dist/main.js' & tsc --noCheck --watch & yarn inspector & shx chmod +x dist/*.js",
```

Comment:
Is it intended that all these commands should run in the background (`&`) instead of sequentially (`&&`)?

Reply:
Yes – the `tsc --watch` is recompiling the TS files to JS when the source changes, and `chokidar` is setting `main.js` to be executable when it's recreated. So these two need to keep watching to reactively change files. Then the inspector is the UI client for debugging